frontend/src/views/RedirectLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react'
+import { useHistory, useLocation } from 'react-router-dom';
+
+// We must have an allowlist of redirects to prevent an attacker from arbitrarily opening external sites.
+const allowedRedirects = new Map([
+	["civhacks", "https://www.civhacks.com"],
+	["civhacks-register", "https://pzudnhexo5v.typeform.com/to/demEictZ"]
+])
```

Comment:
This was a good call because I have tested exploits websites before that didn't have an allowlist for redirects like on typeracer.com haha